src/mscorlib/corefx/System/Globalization/CultureData.Unix.cs

```diff
@@ -180,8 +198,10 @@ private static CultureInfo GetUserDefaultCulture()
 
         private static bool IsCustomCultureId(int cultureId)
```

Comment:
This method will always return false for now, as custom cultures not currently supported. The implementation below is copied from win32